app/assets/javascripts/base.js.erb

```diff
@@ -245,6 +245,25 @@ var instantClick
     getImageForLink(a);
   }
 
+  function focusListener(e) {
```

Comment:
It would be great to explain what you mentioned in the PR as an inline comment to the function to answer the why we need this :)